backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -456,17 +452,24 @@ public void checkNewCameras(ArrayList<Camera> cameras, ArrayList<String> macs)
 
   @Override
   public void deleteCamera(int cameraID) {
-    Statement statement = null;
+    PreparedStatement statement = null;
```

Comment:
This method is becoming quite long now, maybe you can create a method to reduce the amount of code duplication.